controllers/api.js

```diff
@@ -1443,3 +1443,282 @@ exports.getTrakt = async (req, res, next) => {
+
+      // Experimental Properties using PUG-View API
+      fetch(`https://pubchem.ncbi.nlm.nih.gov/rest/pug_view/data/compound/${aspirinCID}/JSON?heading=Physical%20Description`)
+        .then((res) => res.json())
+        .catch((err) => {
+          console.error('Physical Description API error:', err);
+          return { error: 'Failed to fetch physical description' };
+        }),
+
+      fetch(`https://pubchem.ncbi.nlm.nih.gov/rest/pug_view/data/compound/${aspirinCID}/JSON?heading=Color%2FForm`)
+        .then((res) => res.json())
+        .catch((err) => {
+          console.error('Color/Form API error:', err);
+          return { error: 'Failed to fetch color/form' };
+        }),
+
+      fetch(`https://pubchem.ncbi.nlm.nih.gov/rest/pug_view/data/compound/${aspirinCID}/JSON?heading=Odor`)
+        .then((res) => res.json())
+        .catch((err) => {
+          console.error('Odor API error:', err);
+          return { error: 'Failed to fetch odor' };
+        }),
+
+      fetch(`https://pubchem.ncbi.nlm.nih.gov/rest/pug_view/data/compound/${aspirinCID}/JSON?heading=Boiling%20Point`)
+        .then((res) => res.json())
+        .catch((err) => {
+          console.error('Boiling Point API error:', err);
+          return { error: 'Failed to fetch boiling point' };
+        }),
+
+      fetch(`https://pubchem.ncbi.nlm.nih.gov/rest/pug_view/data/compound/${aspirinCID}/JSON?heading=Melting%20Point`)
+        .then((res) => res.json())
+        .catch((err) => {
+          console.error('Melting Point API error:', err);
+          return { error: 'Failed to fetch melting point' };
+        }),
+
+      fetch(`https://pubchem.ncbi.nlm.nih.gov/rest/pug_view/data/compound/${aspirinCID}/JSON?heading=Flash%20Point`)
+        .then((res) => res.json())
+        .catch((err) => {
+          console.error('Flash Point API error:', err);
+          return { error: 'Failed to fetch flash point' };
+        }),
+
+      fetch(`https://pubchem.ncbi.nlm.nih.gov/rest/pug_view/data/compound/${aspirinCID}/JSON?heading=Solubility`)
+        .then((res) => res.json())
+        .catch((err) => {
+          console.error('Solubility API error:', err);
+          return { error: 'Failed to fetch solubility' };
+        }),
+
+      fetch(`https://pubchem.ncbi.nlm.nih.gov/rest/pug_view/data/compound/${aspirinCID}/JSON?heading=Density`)
+        .then((res) => res.json())
+        .catch((err) => {
+          console.error('Density API error:', err);
+          return { error: 'Failed to fetch density' };
+        }),
+
+      fetch(`https://pubchem.ncbi.nlm.nih.gov/rest/pug_view/data/compound/${aspirinCID}/JSON?heading=Vapor%20Pressure`)
+        .then((res) => res.json())
+        .catch((err) => {
+          console.error('Vapor Pressure API error:', err);
+          return { error: 'Failed to fetch vapor pressure' };
+        }),
+    ]);
+
+    // Search for similar compounds
```

Comment:
We can probably drop similar compounds to reduce the code.

Reply:
Yes, dropped the similar compunds section